src/components/App/Header.js

```diff
@@ -0,0 +1,60 @@
+import React, { Component } from 'react'
+import { Link } from 'react-router-dom'
+import classnames from 'classnames'
+import './style.css'
+import IconRandom from '../ui/IconRandom.js'
+import IconFilter from '../ui/IconFilter.js'
+import IconPerson from '../ui/IconPerson.js'
+import IconInfo from '../ui/IconInfo.js'
+
+export default class Header extends Component {
+  render() {
+    const loginLink = this.props.loggedIn ? 'authout' : 'auth'
+    const loginText = this.props.loggedIn ? 'Logout' : 'Login'
+
+    return (
+      <header className="Header" role="banner">
+        <div className="f f-justifyBetween">
+          <div className="App-logo f f-alignSelfCenter">
+            <a href="#">
+              Randomnote
+              <IconRandom />
+            </a>
+          </div>
+          <div className="f Header-buttons">
+            <div>
+              <a onClick={this.props.notebooksToggle}>
+                <button>
+                  <IconFilter />Filter
+                </button>
```

Comment:
Wondering if we could have a single `<SvgIconButton>` component, that accepts props of `width`, `height`, `title`, `iconPath`, `fillPath`.